pages/labTwo.html

```diff
@@ -1,39 +1,170 @@
-<!DOCTYPE html>
-<html lang="en">
+<!DOCTYPE HTML>
+<html lang="en-US">
 <head>
     <meta charset="UTF-8">
-    <title>Lab Project 2</title>
-    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Droid+Serif|Lobster" />
-    <style>
+    <title>Patient Information Site</title>
+    <style type="text/css">
```

Comment:
You should probably use a consistent stylesheet like Bootstrap to avoid inconsistencies; I noticed that the borders of the input fields are getting cut off in Chrome by some of the styles you are applying.